clarity/src/vm/mod.rs

```diff
@@ -86,6 +87,11 @@ use crate::vm::types::{PrincipalData, TypeSignature};
 pub use crate::vm::version::ClarityVersion;
 
 pub const MAX_CALL_STACK_DEPTH: usize = 64;
+pub const MAX_EXECUTION_TIME_SECS: u64 = 30;
+
+#[cfg(test)]
+static TEST_MAX_EXECUTION_TIME: std::sync::Mutex<Duration> =
```

Comment:
You can use `TestFlag` for this which would make it easier to work with!

Reply:
yep, i got some problem in using it with the clarity module, given that the approach looks solid i can start cleaning it up